archivers/libzzip/Portfile

```diff
@@ -64,14 +57,43 @@ post-destroot {
 }
 
 variant sdl description {Enable SDL support} {
-    depends_lib-append      port:libsdl
+    depends_lib-append      port:libsdl2
 
-    configure.args-append   --enable-sdl
+    configure.args-append   -DZZIPSDL=ON
 
     post-destroot {
         xinstall -m 0644 ${worksrcpath}/docs/README.SDL ${destroot}${docdir}
     }
 }
 
-test.run        yes
-test.target     check
+variant python35 conflicts python36 python37 python38 python39 description {Build using Python 3.5} {
+    depends_lib-append \
+        port:python35
+    configure.args-append -DPYTHON_EXECUTABLE="${prefix}/bin/python3.5"
```

Comment:
There is no need for quotation marks here.

Reply:
Couldn't somebody theoretically install macports to a prefix containing a space? Or does macports/Tcl handle those cases automatically?

Reply:
I would guess at least 99% of ports could not build if the prefix contained a space, so I wouldn't bother making the portfile less readable to attempt to accommodate that hypothetical situation.

Reply:
Fair enough.